include/mbedtls/ssl.h

```diff
@@ -3741,39 +3741,50 @@ void mbedtls_ssl_get_dtls_srtp_negotiation_result( const mbedtls_ssl_context *ss
 
 /**
  * \brief          Set the maximum supported version sent from the client side
- *                 and/or accepted at the server side
- *                 (Default: MBEDTLS_SSL_MAX_MAJOR_VERSION, MBEDTLS_SSL_MAX_MINOR_VERSION)
+ *                 and/or accepted at the server side.
  *
- * \note           This ignores ciphersuites from higher versions.
- *
- * \note           With DTLS, use MBEDTLS_SSL_MINOR_VERSION_3 for DTLS 1.2
+ *                 See also the documentation of mbedtls_ssl_conf_min_version().
  *
- * \note           After the handhsake, you can call
- *                 mbedtls_ssl_get_version_number() to see what version was
- *                 negotiated.
+ * \note           This ignores ciphersuites from higher versions.
```

Comment:
While at it, I think this note is out of the picture thus I would just remove it.

Reply:
I wasn't sure what this was supposed to mean, didn't want to further delay this PR with preexisting stuff, and preferred to leave it in and clarify it later™.

Reply:
(It could mean “versions with no available cipher suites will not be considered acceptable even if they're in range”, or on the contrary “the fact that some versions have no available cipher suites will be ignored, so they will be considered acceptable nevertheless”, or possibly something else. I think the second case is what happened, or at least that was the case with 1.0/1.1/1.2, but maybe this has changed now that it's 1.2/1.3?)